openedxscorm/tests.py

```diff
@@ -255,68 +255,3 @@ def test_scorm_get_lesson_score(self, value):
 
         self.assertEqual(response.json, {"value": 20})
 
-    @data(
```

Comment:
Please add few tests to makes sure we have student related info in `student_view`. May be modify existing test ` test_student_view_data`.

Reply:
I don't think the existing `test_student_view_data` tests `student_view` instead a deprecated function named `student_view_data`. But I guess we can create a new test case for `student_view`.